pkg/eve/runme.sh

```diff
@@ -172,6 +188,19 @@ do_sbom() {
   cat /bits/*.spdx.json >&3
 }
 
+prepare_for_platform() {
+    case "$PLATFORM" in
+    imx8mq_evk) #shellcheck disable=SC2039
```

Comment:
- There are some indentation issues (mixing tabs vs spaces)
- We can generalize the platform in the case by using "imx8m*", so it should work for any imx8m platform: https://github.com/rene/eve/commit/ef67818a1e6be51710bca997fbb0617cbfed9f22#diff-18311d3cce6e7a4236ebb9359d68b39c70608f25ef105155a2cb54f9ba36be25L193